### PR TITLE
 overhaul copy plus styles tweaks 

### PR DESCRIPTION
**Pull request by patrickbower**

### CODE REVIEW

File: src/components/team.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          Members from Client
+        </h4>
+        <ul className="team-no-bullets">
+          <li>Product Owner</li>
+          <li>Business Analyst</li>
+        </ul>
+      </div>
+      <div className="col team-border-left">
+        <h4 className="grav-u-font-size-minus-2 team-members">
+          Members from Buildit
```
Comment (danbull) [inline src/components/team.js]: Some members could come from other parts of Wipro?

File: src/components/team.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+const Team = () => (
+  <section className="grav-o-full-bleed__content grav-u-pt-xxl">
+    <h3 className="grav-u-text-centered grav-u-font-size-plus-4">
+      Better Together
+    </h3>
+    <div className="grav-u-text-centered">
+      <p className="lead-text lead-text-large grav-u-pb-xl">
+        Embedded as a small, agile, and cross skilled team we will understand,
```
Comment (danbull) [inline src/components/team.js]: cross-skilled

File: src/components/contact.js [hunk 3]
```diff
@@ -3,15 +3,18 @@ import React from "react";
 const Contact = () => (
   <div className="contact">
     <section className="grav-o-full-bleed__content grav-u-pt-xxl grav-u-pb-xxl">
-      <h2 className="grav-u-text-centered">Engage with us fast.</h2>
-      <p>
-        Messages come direct to us as a team so we endeavour to answer any questions and help with any requests very quickly. 
-      </p>
-      <button className="grav-u-font-size-plus-3"><span role="img" aria-label="finger pointing right">&#x26A1;</span> Contact us</button>
+      <h2 className="grav-u-text-centered">Lets talk</h2>
```
Comment (danbull) [inline src/components/contact.js]: Let's

File: src/components/outcomes.js [hunk 4]
```diff
@@ -3,31 +3,31 @@ import Tick from "../images/tick.svg"
...
+            <p>Pinpoint and answer the unknown, important questions.</p>
           </div>
         </div>
       </li>
       <li className="col-md-4 outcome-list-item">
         <div class="media">
           <img src={Tick} alt="" className="outcome-tick-icon"/>
           <div>
-            <p>Lean but impactful prototypes that are tested with users.</p>
+            <p>Run lean impactful experiments tested direct with users.</p>
```
Comment (danbull) [inline src/components/outcomes.js]: lean, impactful
directly

File: src/components/hero.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ const Hero = () => (
     </svg>
       <h1 className="hero-heading">Cycle 7</h1>
       <p className="grav-u-font-size-plus-2">
-        A human-centred, rapid prototyping process broken down into <strong>3 phases</strong> of activity over <strong>7 weeks</strong>.
+        A rapid innovation process guaranteed to deliver
```
Comment (danbull) [inline src/components/hero.js]: How about "A rapid innovation process guaranteed to deliver real value in 7 weeks." ?

File: src/components/outcomes.js (hunk 4 above)
Comment (patrickbower) [reply src/components/outcomes.js]: directly would be better but I don't think a comm helps - feels odd to pause after the second work

**Pull request merged**